app/components/schools/ect_induction_details_component.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Schools
+  class ECTInductionDetailsComponent < ViewComponent::Base
+    include TeacherHelper
+
+    def initialize(ect)
+      @ect = ect
+    end
+
+    def call
+      safe_join([
+        tag.h2('Induction details', class: 'govuk-heading-m'),
+        govuk_summary_list(rows:)
+      ])
+    end
+
+  private
+
+    def rows
+      [
+        appropriate_body_row,
+        induction_start_date_row
+      ]
+    end
+
+    def appropriate_body_row
+      { key: { text: 'Appropriate body' }, value: { text: @ect.school_reported_appropriate_body_name } }
+    end
+
+    def induction_start_date_row
+      date = induction_start_date
+      if date.present?
+        {
+          key: { text: 'Induction start date' },
+          value: { text: formatted_induction_start_date(date) }
+        }
+      else
+        {
+          key: { text: 'Induction start date' },
+          value: { text: 'Yet to be reported by the appropriate body' }
+        }
+      end
+    end
+
+    def induction_start_date
+      @ect.teacher.first_induction_period&.started_on&.to_fs(:govuk)
+    end
+
+    def formatted_induction_start_date(date)
+      "#{date}<br/><span class=\"govuk-hint\">This has been reported by an appropriate body</span>".html_safe
```

Comment:
When building in code it's probably cleaner to do:

```ruby
safe_join([
  date,
  tag.br,
  tag.span("This has been reported by an appropriate body", class: 'govuk-hint')
])
```

I think the name isn't quite right too, this is adding an informative suffix rather than formatting.